debugger/debugger-worker/src/debugger.csproj

```diff
@@ -19,7 +19,7 @@
     <Compile Include="..\..\..\resharper\build\generated\Model\DebuggerWorker\**\*" LinkBase="Model" Condition="!$(InternalBuild)" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net472">
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3">
```

Comment:
Why has this lost the `.net472` bit?

Reply:
I think this also needs to be added to `PackagesLock.targets`, or we'll get errors in the monorepo

Reply:
This change fixes error
```
       /home/runner/work/resharper-unity/resharper-unity/resharper/resharper-yaml/src/Yaml.Rider/Yaml.Rider.csproj : error NU1605:  JetBrains.ReSharper.Plugins.Yaml.Rider -> Microsoft.NETFramework.ReferenceAssemblies 1.0.3 -> Microsoft.NETFramework.ReferenceAssemblies.net472 (>= 1.0.3)  
```
see https://github.com/JetBrains/resharper-unity/actions/runs/4576761611/jobs/8081322066